controls/radmap/how-to/howto-wraparound-virtualization-visualizationlayer.md

```diff
@@ -0,0 +1,383 @@
+---
+title: Wraparound with Virtualization in VisualizationLayer
+page_title: Wraparound with Virtualization in VisualizationLayer
+description: Wraparound with Virtualization in VisualizationLayer
+slug: radmap-howto-wraparound-virtualization-visualizationlayer
+tags: wraparound,virtualization,visualizationlayer,map
+published: True
+position: 10
+---
+
+# Wraparound with Virtualization in VisualizationLayer
+
+When using wraparound functionality, the virtualization of the RadMap control needs to be set in a more advanced way. 
```

Comment:
"in more advanced way" sounds strange to me. We can discuss the phrase also with Tanya.